src/Services/EmailSender.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity.UI.Services;
+
+namespace Services
+{
+    public class EmailSender : IEmailSender
+    {
+        public Task SendEmailAsync(string email, string subject, string htmlMessage)
+        {
+            return Task.CompletedTask;
+        }
+    }
+}
```

Comment:
Create issue to migrate this code to use [SendGrid](https://app.sendgrid.com/) when we get the API key

Reply:
Done